ignite/cmd/plugin.go

```diff
@@ -180,6 +180,11 @@ func linkPluginHook(rootCmd *cobra.Command, p *plugin.Plugin, hook *plugin.Hook)
 		return
 	}
 
+	c, err := newChainWithHomeFlags(cmd)
+	if err != nil {
+		p.Error = errors.Errorf("unable to obtain chain information.")
+		return
+	}
```

Comment:
@clockworkgr you are right, we should move this back to be initialized before plugin RPC calls, it doesn't make sense to be on top.

Also:

```suggestion
	c, err := newChainWithHomeFlags(cmd)
	if err != nil {
		p.Error = errors.Wrap(err, "unable to obtain chain information")
		return
	}
```